components/brave_wallet/common/common_utils_unittest.cc

```diff
@@ -372,6 +372,17 @@ TEST(CommonUtils, GetNetworkForCardanoAccount) {
                 mojom::AccountKind::kDerived, 123)));
 }
 
+TEST(CommonUtils, IsPolkadotKeyring) {
+  for (const auto& keyring_id : kAllKeyrings) {
```

Comment:
kAllKeyrings needs these new keyrings added. Might also reuqire fixing lots of unrelated tests, but this eventually would be done nevertheless

Reply:
Hmm, it seems like it's more than just fixing tests as I'm hitting a lot of NOTREACHED() stuff as well. There were several `static_assert`s to fix in the test suite but it seems like adding this dramatically increases the scope of the PR and it might best be served by a follow-up PR. I don't mind doing the work, obviously, but it seems like company style is to keep PRs small and focused on a single thing.